src/Microsoft.AspNet.Mvc.Core/MvcBuilderExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.Builder
     /// <summary>
     /// Extension methods for <see cref="IApplicationBuilder"/> to add MVC to the request execution pipeline.
     /// </summary>
-    public static class BuilderExtensions
+    public static class MvcBuilderExtensions
```

Comment:
I didn't find any other builder extensions in this repo outside of these two types... But I'm happy to change this to `MvcApplicationBuilderExtensions` so that it's clearer about the target an less likely to conflict.